src/pretalx/orga/forms/submission.py

```diff
@@ -181,6 +197,17 @@ def save(self, *args, **kwargs):
             task_update_unreleased_schedule_changes.apply_async(
                 kwargs={"event": self.event.slug}
             )
+        if (
+            instance.state in SubmissionStates.accepted_states
```

Comment:
We should drop this check and go through with the deletion in any case. It's a bit wasteful, admittedly, but this way the logic here won't grow out of sync with the central logic regarding which proposals should have slots. Case in point, a while ago we changed the logic so that submissions that are *pending accepted* also have slots. Those proposals would not be covered by the check above.